src/components/molecules/WorldCard/WorldCard.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { faCog, faSignInAlt } from "@fortawesome/free-solid-svg-icons";
+
+import { DEFAULT_VENUE_LOGO } from "settings";
+
+import { World } from "api/admin";
+
+import { WithId } from "utils/id";
+
+import { ButtonNG } from "components/atoms/ButtonNG";
+
+import "./WorldCard.scss";
+
+export interface WorldCardProps {
+  world: WithId<World>;
+}
+
+export const WorldCard: React.FC<WorldCardProps> = ({ world }) => (
+  <div
+    className="WorldCard"
+    style={{
+      backgroundImage: `url(${world.config.landingPageConfig.coverImageUrl})`,
+    }}
```

Comment:
it's worth keeping this calculation in component
```
  const cardVars = useCss({
   backgroundImage: `url(${world.config.landingPageConfig.coverImageUrl})`,
  });

  const cardClasses = classNames("WorldCard", cardVars);
```

and then 
```
<div className={cardClasses}>
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2371/commits/cef8d195f3cdcfef440c04428ffe7f938bbb620f